<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PostBiometricAttendanceWizard</class>
 <widget class="QWizard" name="PostBiometricAttendanceWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Biometric Attendance Wizard</string>
  </property>
  <widget class="QWizardPage" name="wizardPage1">
   <property name="title">
    <string>Select Shift Employees</string>
   </property>
   <property name="subTitle">
    <string>Select the shift and employees to post.</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Attendance Date:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Employees:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Shift:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="cboShift"/>
    </item>
    <item row="1" column="1">
     <widget class="QDateEdit" name="dtpDate">
      <property name="displayFormat">
       <string>dd MMM yyyy</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QTreeWidget" name="trvAttendance">
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>EntryID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>EmployeeID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>#</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date In</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Time In</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date Out</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Time Out</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage2">
   <property name="title">
    <string>Confirm Attendance</string>
   </property>
   <property name="subTitle">
    <string>Confirm the details and then click on 'Save'</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextBrowser" name="txtReview"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="cmdSave">
        <property name="text">
         <string>Save</string>
        </property>
        <property name="icon">
         <iconset resource="rc.qrc">
          <normaloff>:/images/FloppyDisk.png</normaloff>:/images/FloppyDisk.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage">
   <property name="title">
    <string>Finish</string>
   </property>
   <property name="subTitle">
    <string>Click on finish to close this wizard</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>269</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="rc.qrc"/>
 </resources>
 <connections/>
</ui>
